Android/app/src/main/java/io/github/project_travel_mate/destinations/description/FinalCityInfoActivity.java

```diff
@@ -331,25 +331,25 @@ public void slideImages(ArrayList<String> imagesArray) {
 
         for (int i = 0; i < mDotsCount; i++) {
             mDots[i] = new ImageView(this);
-            mDots[i].setImageDrawable(getResources().getDrawable(R.drawable.non_active_dot));
+            mDots[i].setImageDrawable(ContextCompat.getDrawable(this, R.drawable.non_active_dot));
 
             LinearLayout.LayoutParams params = new LinearLayout
                     .LayoutParams(LinearLayout.LayoutParams.WRAP_CONTENT, LinearLayout.LayoutParams.WRAP_CONTENT);
 
             params.setMargins(8, 0, 8, 0);
             sliderDotsPanel.addView(mDots[i], params);
         }
-        mDots[0].setImageDrawable(getResources().getDrawable(R.drawable.active_dot));
+        mDots[0].setImageDrawable(getDrawable(R.drawable.active_dot));
```

Comment:
In line 334 you have used `ContextCompat.getDrawable` but here, you are using `getDrawable` directly. Let's have it consistent in this file.

Reply:
Done i changed it to getDrawable directly